site/package.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "@blockprotocol/site",
-  "version": "0.1.0",
-  "description": "The public-facing blockprotocol.org website",
-  "repository": "https://github.com/blockprotocol/blockprotocol.git",
+  "version": "0.0.0",
```

Comment:
V `0.0.0` is a way of saying ‘this won’t be published on NPM. Turborepo examples to that. I used `0.0.0-local` in the past, but `0.0.0` should work too.